app/src/main/java/org/openobservatory/ooniprobe/activity/ResultDetailActivity.java

```diff
@@ -105,6 +110,54 @@ protected void onResume() {
         load();
     }
 
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.rerun, menu);
+        return super.onCreateOptionsMenu(menu);
+    }
+
+    @Override
+    public boolean onPrepareOptionsMenu(Menu menu) {
+        invalidateOptionsMenu();
+        if (!result.test_group_name.equals(WebsitesSuite.NAME))
+            menu.findItem(R.id.reRun).setVisible(false);
+        return super.onPrepareOptionsMenu(menu);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.reRun:
+                new ConfirmDialogFragment.Builder()
+                        .withExtra("re_run")
+                        //TODO sub these rows when the string is inserted
+                        //.withMessage(getString(R.string.Modal_ReRun_Websites_Title))
+                        //.withPositiveButton(getString(R.string.Modal_ReRun_Websites_Run))
```

Comment:
What value does this commented out lines add?

Reply:
I cant use the `R.string.Modal_ReRun_Websites_Title` asset as it's not present in the code and travis will fail. I will decomment these rows once the  PR is approved and I update the string files

Reply:
Can I see the final PR rather than one where there is magic commented out code?

Reply:
Last time we agreed to add strings files only when the PR is approved to have it more readable to you, but if you want I'll add strings here and then let you know

Reply:
I guess it's not black and white: in this case it's more useful to see a finished PR. Where the strings have mostly a cosmetic value, then it's probably best to defer adding them, as you say.

Reply:
Ok will ping you as soon as I add the strings

Reply:
Updated